system/Helpers/inflector_helper.php

```diff
@@ -93,7 +93,7 @@ function singular(string $string): string
 					'/(s)tatuses$/'														 => '\1\2tatus',
 					'/(c)hildren$/'														 => '\1\2hild',
 					'/(n)ews$/'															 => '\1\2ews',
-					'/([^us])s$/'														 => '\1'
+					'/([^us])s$/'														 => '\1'				
```

Comment:
You've got a couple of places here with whitespace at the end of a line. Our styleguide frowns on that. Please remove.